### PR TITLE
📖 Modify commands & add notes to cronjob-tutorial 

### PR DESCRIPTION
**Pull request by ycyaoxdu**
Fixes [#2439](https://github.com/kubernetes-sigs/kubebuilder/issues/2439)

Add flag to kind command to make it clearer.
Add note for docker push/pull commands. 

Signed-off-by: ycyaoxdu <yaoyuchen0626@163.com>

### CODE REVIEW

**Comment by ycyaoxdu** [comment]
I signed it

File: docs/book/src/cronjob-tutorial/running-webhook.md [hunk 1]
```diff
@@ -19,14 +19,14 @@ You need to follow [this](./cert-manager.md) to install the cert manager bundle.
 Run the following command to build your image locally.
 
 ```bash
-make docker-build
+make docker-build <project-name>:tag
```
Comment (camilamacedo86) [inline docs/book/src/cronjob-tutorial/running-webhook.md:22]:
```suggestion
make deploy IMG=<some-registry>/<project-name>:tag
```

**Review by camilamacedo86 — COMMENTED** [review]
Thank you for the contribution 🥇  
Just a nit. Otherwise, lgtm

**Review by camilamacedo86 — APPROVED** [review]
👍

**Pull request merged**